Road/notebooks/feature_selected.ipynb

```diff
@@ -0,0 +1,1196 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    from source.features import feature_selected as FS
</pre><p>Could you please use a representative name instead of <code>FS</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Road-Status/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>